elliottlib/bzutil.py

```diff
@@ -365,6 +365,9 @@ def create_bug(self, title, description, target_status, keywords: List) -> Bugzi
             keywords=keywords,
             description=description)
         new_bug = self._client.createbug(create_info)
+        if noop:
```

Comment:
Seems to be too late to check noop. Bug is already created when this line is reached.